NuGet.config

```diff
@@ -7,15 +7,8 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-standard-a5b5f2e" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-standard-a5b5f2e1/nuget/v3/index.json" />
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
-
-    <!--
-      'src/test/PrepareTestAssets/PrepareTestAssets.proj' generates a NuGet.config file using this
-      one as a template. The following line is a marker to insert the test restore sources.
-    -->
-    <!-- TEST_RESTORE_SOURCES_INSERTION_LINE -->
```

Comment:
The change to stop Darc from removing these should be rolled out, but @wtgodbe, I think because you had to add the `dotnet3` feed, it would make sense to fix this PR up the rest of the way to unblock future PRs which will (?) also need it.

Reply:
@dagood what else needs to be done here? An Arcade update?

Reply:
Just adding back the comments here will work. For future PRs, this will just work, because Darc does its thing serverside, not based on anything in the repo.